internal/gqlclient/gqlclient.go

```diff
@@ -107,6 +109,11 @@ func (c *Client) RunWithContext(ctx context.Context, request Request, response i
 	if err != nil {
 		return NewRequestError(err, request)
 	}
+	responseData, err := json.MarshalIndent(response, "", "  ")
+	if err != nil {
+		return errs.Wrap(err, "failed to marshal response")
+	}
+	logging.Debug("gqlclient: response: %s", responseData)
```

Comment:
Would be good if this used the `ACTIVESTATE_DEBUG_SERVICE_REQUESTS` and only logged it when set. Otherwise, this'll get a bit verbose and we will end up missing valuable data in rollbar logs as they only support a limited log size.